### DISCUSSION TITLE
Can't move device from one group to another

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5531] [General]
As described "nearly" fresh install, add some groups, some devices in and now one device (with an agent) should be in "OS with agent" but it never appeared there, so i add in an other group without agent. Than i want to move in the tight group but this won't work. No Message or something else. 
What could i check?

**Comment by Ylianst** [comment]
You can't move devices between device groups of different types. So, a agent device group can move devices to another agent device group. Hope that helps.
